pkg/daemon/controller/controller.go

```diff
@@ -477,16 +537,65 @@ func (c *Controller) iptablesSyncLoop() {
 				iptablesManager := c.getIPtablesManager(subnet.Spec.Range.Version)
 				iptablesManager.RecordSubnet(cidr, ramav1.GetNetworkType(network) == ramav1.NetworkTypeOverlay)
 			}
+		}
+
+		if daemonfeature.MultiClusterEnabled() {
+			remoteSubnetList, err := c.remoteSubnetLister.List(labels.Everything())
+			if err != nil {
+				return fmt.Errorf("list remote network failed: %v", err)
+			}
 
-			if err := c.iptablesV4Manager.SyncRules(); err != nil {
-				return fmt.Errorf("sync v4 iptables rule failed: %v", err)
+			var remoteOverlayExist bool
+			for _, remoteSubnet := range remoteSubnetList {
+				if ramav1.GetRemoteSubnetType(remoteSubnet) == ramav1.NetworkTypeOverlay {
+					remoteOverlayExist = true
+					break
+				}
 			}
 
-			if err := c.iptablesV6Manager.SyncRules(); err != nil {
-				return fmt.Errorf("sync v6 iptables rule failed: %v", err)
+			// add remote overlay subnets
+			if remoteOverlayExist {
+				// Record remote vtep ip.
+				vtepList, err := c.remoteVtepLister.List(labels.Everything())
```

Comment:
If only remote underlay subnet exist, we also need to update node ip addresses and remote subnet to iptables manager.